functions.js

```diff
@@ -1 +1,45 @@
-// YOUR CODE HERE
+// FUNCTION LAB HW
+
+//1.
+function maxOrMin(num1, num2, max){
+    if (max === true){
+        return(num1 > num2 ? num1 : num2);
+    }
+    else if (max === false){
+        return(num1 < num2 ? num1 : num2);
+    }
+    else {
+        console.log("error");
+    }
+}
+
+//2.
+var sampleArray = [1, 2, ,3, 4, 5 ,6, 7];
+
+function sillySum(arr){
+    return arr.map(function callBack(element, index){
+        return element + (element * index)
+    });
+}
+
+sillySum(sampleArray);
+
```

Comment:
should return the sum of the values in the array after they have been multiplied by their index in the original array.  Good use of the callback though